host/greatfet/commands/greatfet_shell.py

```diff
@@ -69,7 +68,12 @@ def main():
 
 
     # Create a new shell, and give it access to our created GreatFET object.
-    shell = TerminalInteractiveShell()
+    # Apr 18 2023 - Changed method for creating the shell, to support changes in IPython v8.11+
+    #             - ref. https://github.com/ipython/ipython/issues/13966
```

Comment:
You've already done a great job of documenting this in the PR itself so it doesn't need to be in the code as well.

Reply:
Updated the PR, comment removed.